tests/benchmarks/longmemeval/test_longmemeval.py

```
@@ -0,0 +1,746 @@
+                    if memory_id:
+                        requests.delete(
+                            f"{self.config.base_url}/memory/{memory_id}",
+                            headers=self.headers,
+                            timeout=self.config.request_timeout,
+                        )
+                        total_deleted += 1
+
+                if len(results) < 100:
+                    break
+
+            except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)